UnrealPlugin/Source/PlayfabGSDK/Private/PlayfabGSDK.cpp

```diff
@@ -75,7 +83,6 @@ bool FPlayFabGSDKModule::ReadyForPlayers()
 	if (GSDKInternal->GetHeartbeatRequest().CurrentGameState != EGameState::Active)
 	{
 		GSDKInternal->SetState(EGameState::StandingBy);
-		GSDKInternal->GetTransitionToActiveEvent()->Wait();
```

Comment:
Wouldnt it be useful to still return whether Server is Ready for Players?

Reply:
In the case of blueprints: When the delegate, or the blueprint event is triggered, then it should be active. IE, always true. There's no need for an input that is always the same value. If it wasn't true, it wouldn't fire. So repeating myself a lot, I think the delegates/events are void(void), and this method seems to be meaningless as every place where it was used is irrelevant now.

Reply:
I think the function is still relevant as it sets the server into the standby state if not already active which is necessary, because otherwise the server is stuck in the initializing state

Reply:
Aha, this is an interesting development. Ok. We should definitely separate the methods with intent. One method for the customer's gameinstance to say "I'm ready". Probably something like "SetGameInitializationComplete".

The other communication was the return value, which I have been saying to remove entirely, and let the delegate perform that work. So yes, we need to keep a method, but the name has to indicate what it does. It is the game telling the GSDK that the game is ready to go.

Thanks strwork.